### ISSUE TITLE
Specify "argument should not be null" exception

### ISSUE THREAD

**Issue by Kira-Cesonia** [javers/javers#996]
Frontend has handed me the following error stemming from our new Javers implementation:

```
Exception in thread "Thread-143" java.lang.IllegalArgumentException: argument should not be null
	at org.javers.common.validation.Validate.argumentsAreNotNull(Validate.java:33)
	at org.javers.core.JaversCore.commit(JaversCore.java:80)
	at org.javers.core.JaversCore.commit(JaversCore.java:69)
```

The problem is that this error does not happen in the backend tests that I have, and just from the error message, I can't say which of the possible arguments is null.

It would be great if the error message could specify which of the arguments was null.

**Comment by bartoszwalacik** [comment]
agreed, contribute a PR, we will merge it

**Comment by bartoszwalacik** [comment]
fixed in 5.10.3
